package.json

```diff
@@ -43,16 +54,5 @@
     "nyc": "^14.1.1",
     "semantic-release": "^18.0.1",
     "tape": "^4.2.2"
-  },
```

Comment:
@v-menzhou, I have adjusted the package.json file according to your suggestions, could I get green lights to pass this PR